src/components/panel/panel.js

```diff
@@ -1950,25 +1954,36 @@ MdPanelPosition.prototype._calculatePanelPosition = function(panelEl, position)
   var targetRight = targetBounds.right;
   var targetWidth = targetBounds.width;
 
+  function reverseXPosition (position) {
+    var start = 'start';
+    var end = 'end';
+
+    if (position.includes(start)) {
```

Comment:
String.prototype.includes definitely looks nice, but according to [MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes) it's not supported in IE at all. I think it's better to use `position.indexOf(start) > -1` here.